collect_app/src/main/java/org/odk/collect/android/formentry/SwipeHandler.kt

```diff
@@ -90,9 +91,17 @@ class SwipeHandler(context: Context, generalSettings: Settings) {
                 if (diffX > xpixellimit && diffY < ypixellimit || diffX > xpixellimit * 2) {
                     beenSwiped = true
                     if (e1.x > e2.x) {
-                        onSwipe.onSwipeForward()
+                        if (view!!.context.isLTR()) {
```

Comment:
This `!!` jumped out at me but it's no worse than what's there now -- line 73 already does it. It's a bit weird that two conditionals above (and below the first uses of `view!!`) have not-null assertions.

Reply:
Yeah, I didn't like it too... I could use one of the extension functions that assert it's not null but that would mean one more nested statement so I don't know what's better. I think for now it's ok as it is.